src/style.css

```diff
@@ -1,3 +1,59 @@
 body {
   margin: 0;
+  background-color: rgb(154, 154, 154);
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.header {
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.nav {
+  display: flex;
+  background: #fff;
+  height: 60px;
+  justify-content: flex-end;
+  align-items: center;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+.nav__link {
+  text-decoration: none;
+  padding: 20px;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+  padding: 22.5px;
+  padding-left: 0;
+  padding-right: 0;
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  background-color: #00acdc;
+  height: 4px;
+  border-radius: 8px;
+  right: 0;
+  left: 0;
+  position: absolute;
+  bottom: 0;
+}
+
+.img {
+  position: absolute;
+  left: 50px;
+  top: 10px;
+}
+
+.l:hover {
+  color: #00acdc;
+  padding: 22.5px;
+  padding-left: 0;
+  padding-right: 0;
```

Comment:
The `:hover` pseudo-class should not alter the layout dimensions like padding, as it can cause layout shifts. Consider changing the visual appearance using other properties such as color, background-color, or border. If you need to adjust the size, consider using transform scale instead.